gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/ae_datastore.rb

```diff
@@ -0,0 +1,6 @@
+class ManageIQ::Providers::Amazon::AeDatastore
+  def self.seed
+    dir = Rails.root.join("gems/manageiq-providers-amazon/db/fixtures/ae_datastore/")
+    MiqAeDatastore.reset_domain(dir, 'Amazon', Tenant.root_tenant)
```

Comment:
@gmcculloug this is just a prove, that it works.
Is this is the way to go via seeding?
And should it be in this new class?